src/components/Footer/footer.css

```diff
@@ -20,8 +20,13 @@ footer .links {
   justify-content: center;
```

Comment:
use makeStyles so it is consistent with the rest of the project!

Reply:
aab2f515ac81bdf63b8bef8963ea1d809f164d2f